content/sensu-go/5.18/release-notes.md

```diff
@@ -53,6 +54,28 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 5.18.0 release notes
+
+**February 24, 2020** &mdash; The latest release of Sensu Go, version 5.18.0, is now available for download.
+**PLACEHOLDER FOR RELEASE SUMMARY**
+
+See the [upgrade guide][1] to upgrade Sensu to version 5.18.0.
+
+**IMPROVEMENTS:**
+
+- The `event.entity.entity_class` value now defaults to `proxy` for [`POST /events`][114] requests.
+- Proxy entities are now automatically created when events are published with an entity that does not exist.
+- Supported Bonsai asset versions are now prefixed with the letter `v` (for example, `v1.2.0`).
```

Comment:
Actually sensuctl now *also* accept Bonsai asset versions prefixed with `v`; most Bonsai assets will continue to use the non-prefixed version.

Reply:
Is my suggestion below correct?